docs/analyzers/Analyzer Samples.md

```diff
@@ -0,0 +1,30 @@
+﻿**Samples Location:**
+
+[This](..//..//src//Diagnostics//Samples) folder contains sample C# and VB analyzers to demonstrate recommended implementation models for different analysis scenarios.
```

Comment:
I had couple of follow up questions on that:

1. Do we want to move the added projects into Samples.sln or still keep them as separate solutions? All projects in Sample.sln are built against the local bits of Microsoft.CodeAnalysis assemblies from output folder, while the analyzer samples are created directly from our template project, and hence have nuget references to CodeAnalysis.

2. Alternatively, I can do both: Retain the CSharpAnalyzers.sln and BasicAnalyzer.sln added here (but moved to src\Samples directory), and also add the individual projects from it to Sample.sln.